.github/workflows/cypress-test-tenancy-disabled.yml

```diff
@@ -83,4 +83,4 @@ jobs:
           git clone https://github.com/opensearch-project/opensearch-dashboards-functional-test.git
           cd opensearch-dashboards-functional-test
           npm install cypress --save-dev
-          yarn cypress:run-with-security --browser chrome --spec "cypress/integration/plugins/security-dashboards-plugin/inaccessible_tenancy_features.js"
+          yarn cypress:run-with-security --browser chrome --spec "cypress/integration/plugins/security/inaccessible_tenancy_features.js"
```

Comment:
This should be addressed in the FTR repo. This file is in `security/` on `main`, but `security-dashboards-plugin/` on 2.x. Should this be moved to `security-dashboards-plugin` on main in the FTR repo?

Reply:
Let's address this separately?

Reply:
The best path forward I see is: 
- Get all tests working and consolidated under `/security` folder, since this is the current folder integration tests are looking at for autocuts
- If we determine it to be necessary to change to `/security-dashboards-plugin` to align with other plugins, we can make that change in both FTR and the build repo